recipes/wg21-linear_algebra/all/conanfile.py

```diff
@@ -35,39 +35,42 @@ def _minimum_compilers_version(self):
             "apple-clang": "11"
```

Comment:
This is missing the msvc compiler which should be 192 afaik

Reply:
Thanks